app/gui2/src/stores/graph/graphDatabase.ts

```diff
@@ -138,6 +167,37 @@ export class GraphDb {
         this.nodes.delete(nodeId)
       }
     }
+
+    const nodeRects = [...nodeRectMap.values()]
+    const rectsHeight =
+      numberOfUnpositionedNodes * (theme.node.height + theme.node.vertical_gap) -
+      theme.node.vertical_gap
+    const { position: rectsPosition } = nonDictatedPlacement(
+      new Vec2(maxUnpositionedNodeWidth, rectsHeight),
+      {
+        nodeRects,
+        // The rest of the properties should not matter.
+        selectedNodeRects: [],
+        screenBounds: Rect.Zero,
+        mousePosition: Vec2.Zero,
+      },
+    )
+    let nodeIndex = 0
+    for (const nodeId of this.allNodeIds()) {
+      const meta = getMeta(nodeId)
+      if (meta) continue
+      const node = this.nodes.get(nodeId)!
+      // The node with currently must be fixed
```

Comment:
typo

Reply:
Ah, that comment is out of date too. Nice catch, thanks!